resources/ips/doc-transform.erb

```diff
@@ -0,0 +1,2 @@
+<transform dir path=<%= pathdir %>$ -> edit group bin sys>
+<transform file depend -> edit pkg.debug.depend.file ruby env>
```

Comment:
The IPS package creation evaluates the self executable ruby scripts like "ssh_tunnel_bug.rb" included in omnibus as an external ruby dependency, so this transform file resolves these by marking them as internal environment dependencies in the .res file.  It would not take any effect in a non Ruby project as there would be no dependencies to update.